### ISSUE TITLE
Way to control each single guide with brush tool

### ISSUE THREAD

**Issue by ghost** [ikakupa/medusa-nodes-bug-reports#5]
REQUEST:
Way to control each single guide with brush tool
(For users who use their brush as their main tool)

ISSUE:
The brush tool effects other guides in 3D viewport.
So I have to control the sizes of the brush many times if I need to control each single strand. That is stressful😢

IDEAS:
1.Masking by each single guide of Guide Deformer with touching the guide directly in 3D viewport
2.Switching a group of strand generated and layered in Medusa Node Menu

I want to control strands with a intuitive way, like with brush tool.
I can control strands that are clumped and aligned by Guide Deformer and other Deformers. which is useful and intuitive, I want to enhance the intuitivity.
I believe users, who uses their brush as a main tool, think the same desire to me.😉

**Comment by ikakupa** [comment]
Hello Shaku,

> Way to control each single guide with brush tool

Currently there is an option to sculpt each and every guide which was manually placed by the user. I am probably missing something here.

> The brush tool effects other guides in 3D viewport.

But you can use Masking for that.
![image](https://user-images.githubusercontent.com/64023824/218207707-be7b4bb5-e409-4b18-b52e-2d9f38dcbf69.png)

Just to make it more clear for me. 
Would you like to be able to select hairs that are tied to certain guide curves?
Or would you like to be able to sculpt the hairs after the deformations from the deformers?

**Comment by ghost** [comment]
Hi ikakupa,

>But you can use Masking for that.

Oh, I see that is already in Blender, which is what I want😊


>1. Would you like to be able to select hairs that are tied to certain guide curves?
>2. Or would you like to be able to sculpt the hairs after the deformations from the deformers?


No.2. is close to what I want😊
I would like to use Medusa Node for rough modeling, not for details.
Which means, when I need to sculpt tiny single hair, I would sculpt them after the deformations from the deformers.
(Perhaps I have confused you with my wrong word "a strand", I used it as a meaning of a bundle of hair )
![image](https://user-images.githubusercontent.com/59954150/218240364-48c934d1-fd50-4993-b744-5bc903b0816d.png)

**Comment by ikakupa** [comment]
Yes, I see what you mean it is not so easy to implement. This is something I am working on with the prototype as well. It doesn't need to be some special brush type in my opinion. 

This probably could be done with the some kind of caching system. So when ever you would like to sculpt on top of any deformer, the node tree will be baked down and you would be able to make some kind of manuell adjustments.
